src/components/Docs.js

```diff
@@ -94,7 +94,7 @@ const RelatedLinks = (props) => {
 
 RelatedLinks.propTypes = {
   links: PropTypes.arrayOf(
-    PropTypes.arrayOf(PropTypes.string.isRequired).isRequired
+    PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
```

Comment:
for prs, its best to keep them short and concise. can we undo these formatting changes? it looks like the whole directory was formatted instead of just the files you edited/added (like App.js and CloudDataBaseForm.js).